ExtLibs/MissionPlanner.Drawing/Graphics.cs

```diff
@@ -1606,7 +1606,7 @@ public static Graphics FromHwnd(IntPtr windowHandle)
 
             //get client rect
             //hwnd to hdc
-            return HwndToGraphics(windowHandle);
+            //return HwndToGraphics(windowHandle);
```

Comment:
i question this?

Reply:
This method has two RETURNs in the sequence.
The first RETURN will be executed and this RETURN will not be executed permanently.
I have commented out this RETURN handling as I think it will be useful in the future.